src/clients/Deluge.ts

```diff
@@ -91,22 +114,27 @@ export default class Deluge implements TorrentClient {
 	/**
 	 * ensures authentication and sends JSON-RPC calls to deluge
 	 */
-	private async call(method: string, params: object, retries = 1) {
+	private async call<ResultType>(
+		method: string,
+		params: object,
```

Comment:
we at least know params has to be an array, right? how about any[]

Reply:
I think we might be able to get away with string[] with its current usage, but I'd have to double check every usage. It would potentially be tech debt if we ever expanded the capabilities though and

Reply:
since explicit any[] error's i'm going with your suggestion of unknown[]